File: campeonato-api/src/main/java/com/luizmarcelo/campeonatoapi/model/Time.java
package com.luizmarcelo.campeonatoapi.model;

import jakarta.persistence.*;
import lombok.*;

@Getter
@Setter
@Entity(name = "time")
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Time {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column(name = "nome")
    private String nome;
    @Column(name = "descricao")
    private String descricao;
    @Column(name = "url_foto")
    private String urlFoto;

}

File: campeonato-front/src/app/models/time/time.ts
export class Time {

  id: number;
  nome: string;
  descricao: string;
  urlFoto: string;


  constructor(id: number, nome: string, descricao: string, urlFoto: string) {
    this.id = id;
    this.nome = nome;
    this.descricao = descricao;
    this.urlFoto = urlFoto;
  }
}
